docs/pages/working-with-models.rst

```diff
@@ -1,5 +1,50 @@
-Working with models recipes.
-============================
+Advanced usage
+==============
+
+.. _custom_fallback:
+
+Custom fallback language
+------------------------
+
+By default, fallback is centrally configered with :ref:`settings_fallback`.
+That might not be sufficient, for example if part of the content is created for a single language which is not ``LANGUAGE_CODE``.
+
+In that case, it can be con configured per-record using the ``fallback_language_field`` argument to ``TranslationField``::
+
+    class NewsRoom(models.Model):
+        name = models.CharField(max_length=255)
+        default_language = models.CharField(max_length=2)
+
+        i18n = TranslationField(fields=("name",), fallback_language_field="default_language")
+
+You can traverse foreign key relations too::
+
+    class Article(models.Model):
+        content = models.CharField(max_length=255)
+        newsroom = models.ForeignKey(NewsRoom)
+
+        i18n = TranslationField(fields=("content",), fallback_language_field="newsroom__default_language")
+
+Note that
```

Comment:
```suggestion
Note::
```

Reply:
Volgens mij is het prima zo:

<img width="697" alt="Screenshot 2020-11-23 at 11 04 53" src="https://user-images.githubusercontent.com/1470389/99949700-c38da780-2d7b-11eb-9efe-f1c66b9d2eec.png">